spec/requests/cloud_volumes_spec.rb

```diff
@@ -8,7 +8,7 @@
 # GET /api/cloud_volumes/:id
 #
 
-describe "Cloud Volumes API" do
+fdescribe "Cloud Volumes API" do
```

Comment:
This looks like a typo

Reply:
Hmm... somehow Travis passed 😕

Reply:
Well, it's [a real thing](https://relishapp.com/rspec/rspec-core/v/3-10/docs/filtering/filter-run-when-matching#groups-can-be-focused-with-%60fdescribe%60-or-%60fcontext%60), but I don't think it was intentional.

Reply:
@bdunne @agrare sorry about that. This actually was intentional, but it wasnt supposed to be part of the PR. in order to run specific specs we sometime use[ Inclusion filters](https://relishapp.com/rspec/rspec-core/v/2-6/docs/filtering/inclusion-filters). this is because running a specific file not always works. in short, I wrote a script that add  config.filter_run focus: true to spec_helper.rb (simply sed -i '/RSpec.configure.*/a \ \ config.filter_run focus: true' ./spec/spec_helper.rb) and then only tagged tests will be run. Tagging is done by adding ':focus => true' (or prepend an f) to it, specify, describe or context.
my script also remove the modified spec_helper.rb when it done but not the tagging and I forgot to remove it before the push. this shouldn't append, sorry about that.

Reply:
I removed it now :)